api/utils/utils.py

```diff
@@ -0,0 +1,16 @@
+def verify_data(Tx, data):
+    import requests
+    import json
+    import hashlib
+    import base64
+    r = requests.get(
+        'http://localhost:5000/get_transaction/'+Tx)
```

Comment:
Cloud you please change the `http://localhost:5000` to configuration parameter?

Reply:
Fix in a379c2d.